doc/DESI_SPECTRO_DATA/NIGHT/EXPID/guide-EXPID.rst

```diff
@@ -16,37 +16,39 @@ Contents
 Number EXTNAME  Type             Contents
 ====== ======== ================ ===================
 HDU00_ GUIDER   Empty HDU        Header keywords only
-HDU01_ PMGSTARS BINTABLE         *Brief Description*
-HDU02_ GUIDE5   Compressed IMAGE Guide image 5
-HDU03_ GUIDE5T  BINTABLE         Guide image 5 metadata
-HDU04_ GUIDE3   Compressed IMAGE Guide image 3
-HDU05_ GUIDE3T  BINTABLE         Guide image 3 metadata
-HDU06_ GUIDE2   Compressed IMAGE Guide image 2
-HDU07_ GUIDE2T  BINTABLE         Guide image 2 metadata
-HDU08_ GUIDE7   Compressed IMAGE Guide image 7
-HDU09_ GUIDE7T  BINTABLE         Guide image 7 metadata
-HDU10_ GUIDE8   Compressed IMAGE Guide image 8
-HDU11_ GUIDE8T  BINTABLE         Guide image 8 metadata
-HDU12_ GUIDE0   Compressed IMAGE Guide image 0
-HDU13_ GUIDE0T  BINTABLE         Guide image 0 metadata
+HDU01_ PMGSTARS BINTABLE         PlateMaker star table
+HDU02_ GUIDE5   Compressed IMAGE GUIDE5 image cube
+HDU03_ GUIDE5T  BINTABLE         GUIDE5 image cube metadata
+HDU04_ GUIDE3   Compressed IMAGE GUIDE3 image cube
+HDU05_ GUIDE3T  BINTABLE         GUIDE3 image cube metadata
+HDU06_ GUIDE2   Compressed IMAGE GUIDE2 image cube
+HDU07_ GUIDE2T  BINTABLE         GUIDE2 image cube metadata
+HDU08_ GUIDE7   Compressed IMAGE GUIDE7 image cube
+HDU09_ GUIDE7T  BINTABLE         GUIDE7 image cube metadata
+HDU10_ GUIDE8   Compressed IMAGE GUIDE8 image cube
+HDU11_ GUIDE8T  BINTABLE         GUIDE8 image cube metadata
+HDU12_ GUIDE0   Compressed IMAGE GUIDE0 image cube
+HDU13_ GUIDE0T  BINTABLE         GUIDE0 image cube metadata
 ====== ======== ================ ===================
 
 The GUIDEn data will be 3D[nframes, ny, nx] such that
 ``data[i]`` is the 2D GFA frame number ``i``.  Row ``i`` of the
-GUIDEnT table will contain the metadata about that frame, *e.g.* the
-DATE-OBS and EXPTIME.
+GUIDEnT table will contain the metadata about that frame, for example the
+DATE-OBS and EXPTIME. The image cubes include the overscan and prescan;
+``nx`` is 2248 and `ny` is 1032. The active image area is 2048 by 1032
+pixels once overscan and prescan have been removed.
 
 Note that other than the blank data primary HDU, the order of the other
 HDUs is arbitrary and some GUIDEn(T) HDUs may even be missing.  The
-nominal set (0,2,4,5,7,9) is the plan for full DESI, but particularly during
-commissioning other combinations will appear in the data.
+nominal set (0,2,3,5,7,8) is the plan for full DESI, but particularly during
+commissioning other subsets will appear in the data. Also note
+that the acquisition image is frame ``i = 0`` when there is an acquisition
+image (a ``sequence=DESI`` exposure has an acquisition image but a
+``sequence=_Split`` exposure does not, for example).
```

Comment:
Is `_Split` really preceded by an underscore?

Reply:
Yes, the sequence name _Split does start with an underscore.